src/vcpkg/platform-expression.cpp

```diff
@@ -532,6 +534,7 @@ namespace vcpkg::PlatformExpression
                         case Identifier::arm64: return true_if_exists_and_equal("VCPKG_TARGET_ARCHITECTURE", "arm64");
                         case Identifier::windows:
                             return true_if_exists_and_equal("VCPKG_CMAKE_SYSTEM_NAME", "") ||
+                                   true_if_exists_and_equal("VCPKG_CMAKE_SYSTEM_NAME", "Windows") ||
```

Comment:
In the current implementation if you explicitly set ``VCPKG_CMAKE_SYSTEM_NAME`` to "Windows", it wouldn't return true for the 'windows' expression.

Was this intentional or just an assumption that all Windows system usage would be "default"?